package.json

```
@@ -16,9 +16,9 @@
   },
   "dependencies": {
     "@sentry/react": "^7.64.0",
-    "@walletconnect/core": "^2.10.5",
-    "@walletconnect/identity-keys": "^0.2.2",
-    "@walletconnect/notify-client": "^0.14.5",
+    "@walletconnect/core": "2.11.0",
+    "@walletconnect/identity-keys": "^1.0.1",
+    "@walletconnect/notify-client": "^0.16.0-5ff9624",
```

Need to bump to 0.16.0 exact I believe (saw you merged the release PR for notify-client)

```suggestion
    "@walletconnect/notify-client": "0.16.0",
```